src/Linux/Host/Broker/Impl/Services/LinuxRHostProcessService.cs

```diff
@@ -41,15 +45,37 @@ class LinuxRHostProcessService : IRHostProcessService {
             return new UnixProcess(process);
         }
 
+        private string GetUsrLibPath() {
+            const string x64Lib = "/usr/lib/x86_64-linux-gnu";
```

Comment:
MRO has other binaries e.g., libRBlas.so that gets loaded by libR.so. since MRO is in this location `/usr/lib64/microsoft-r/3.3/lib64/R/lib`. I am not sure what is the search rule for dlopen is, and also the EFL for libR.so does not specify where to find the lib relative to libR.so. It just specifies that it is needed.
```
user1@ub-dev:/usr/lib64/microsoft-r/3.3/lib64/R/lib$ readelf -d libR.so

Dynamic section at offset 0x1d6e328 contains 30 entries:
  Tag        Type                         Name/Value
 0x0000000000000001 (NEEDED)             Shared library: [libRblas.so]
 0x0000000000000001 (NEEDED)             Shared library: [libgfortran.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libm.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [librt.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libdl.so.2]
 0x0000000000000001 (NEEDED)             Shared library: [libstdc++.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libgomp.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libgcc_s.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libpthread.so.0]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [ld-linux-x86-64.so.2]
 0x000000000000000c (INIT)               0xda0b0
 0x000000000000000d (FINI)               0x3cde58
 0x000000006ffffef5 (GNU_HASH)           0x158
 0x0000000000000005 (STRTAB)             0x276e0
 0x0000000000000006 (SYMTAB)             0x9b90
 0x000000000000000a (STRSZ)              148201 (bytes)
 0x000000000000000b (SYMENT)             24 (bytes)
 0x0000000000000003 (PLTGOT)             0x1f6ff48
 0x0000000000000002 (PLTRELSZ)           63072 (bytes)
 0x0000000000000014 (PLTREL)             RELA
 0x0000000000000017 (JMPREL)             0xcaa50
 0x0000000000000007 (RELA)               0x4e2c8
 0x0000000000000008 (RELASZ)             509832 (bytes)
 0x0000000000000009 (RELAENT)            24 (bytes)
 0x000000006ffffffe (VERNEED)            0x4e168
 0x000000006fffffff (VERNEEDNUM)         9
 0x000000006ffffff0 (VERSYM)             0x4b9ca
 0x000000006ffffff9 (RELACOUNT)          18466
 0x0000000000000000 (NULL)               0x0

```